django_ses/views.py

```diff
@@ -388,7 +388,11 @@ def post(self, request, *args, **kwargs):
                     'notification': notification,
                 })
             else:
-                event_type = message.get('eventType')
+                # https://docs.aws.amazon.com/ses/latest/DeveloperGuide/notification-contents.html#top-level-json-object
+                # notificationType field: A string that holds the type of notification
+                # represented by the JSON object.
+                # Possible values are Bounce, Complaint, or Delivery.
+                event_type = message.get('eventType') or message.get('notificationType')
```

Comment:
Why keep `eventType` then? Or if we do keep it, should the priority be reversed here?

Reply:
Okay now it's clear why I got different Field name.
There are 2 ways to setup SNS and SES.
1. setup via notification. here only 3 types available: bounce, complaint, delivery http://docs.amazonwebservices.com/ses/latest/DeveloperGuide/NotificationsViaSNS.html
2. setup via events (many types including click, opens and so on) https://docs.aws.amazon.com/ses/latest/DeveloperGuide/monitor-using-event-publishing.html

So in case when we set as notification we have 'notificationType' if we set as event we have 'eventType' field. So now the question is what setup should be supported. Options:
1. support events (they have more types: opens, clicks, bounces and etc). Than we need remove all mentions about notifications from comments that may lead to wrong setup. (Particularly link https://docs.aws.amazon.com/ses/latest/DeveloperGuide/monitor-using-event-publishing.html should be replaced by https://docs.aws.amazon.com/ses/latest/DeveloperGuide/monitor-using-event-publishing.html)
2. support both (then we do need to update documentation and code)

As events have more features probably there is no reason to support both. So only documentation and comments update is needed =)

Reply:
OK, let's do option 1 - support just events. And update docs. Could you make this change? Thanks!

Reply:
Updated link for setup instructions to events page instead of notifications so there will be no confusion because of different AWS response

Reply:
Thank you!